next-app/src/routes/+page.svelte

```diff
@@ -31,93 +32,176 @@
+			<Button on:click={ async () => await GET({url: '//LFAPP'}) } danger>Cause network error</Button>
+			<Button on:click={ () => {throw Error("sorry, that's not a good password")} } danger>Cause biz logic error</Button>
+			<Button on:click={ async () => await GET({url: '//httpbin.org/status/500'}) } danger>Cause backend error</Button>
+
+			<h3>Server</h3>
+			The change password page has a few options:
+			<ul>
+				<li>Submit the form without supplying anything</li>
+				<li>Only supply the password and not the confirm</li>
+				<li>Try to submit the form without being authenticated</li>
+			</ul>
+		</section>
+
+		<section>
+			<h2>UI library <small><a href=//daisyui.com rel=noreferrer target=_blank>//daisyui.com</a></small></h2>
+
+			<h3>Button</h3>
+			<button class='btn btn-primary'>primary</button>
+			<button class='btn btn-secondary'>secondary</button>
+			<button class='btn btn-error'>error</button>
+			<button class='btn btn-lg'>large</button>
+			<button class='btn' disabled>disabled</button>
+
+			<h3>Light/Dark mode</h3>
+			<input type=checkbox bind:checked={ dark_mode } class='toggle toggle-primary toggle-lg'>
+
+			<h3>Stats</h3>
+			<Stats {stats} />
+
+			<h3>Drawer</h3>
+			<label for=left_drawer class='btn btn-primary drawer-button'>Open left drawer</label>
+		</section>
+
+		<section>
+			<h2>Custom components</h2>
+
+			<h3>Button</h3>
+			<Button>primary</Button>
+			<Button danger>error</Button>
+			<Button class='btn-outline'>outline</Button>
+			<Button class='btn-xs sm:btn-sm'>extra small</Button>
+
+			<h3>Details</h3>
+			<Details>
+				<div slot=summary>
+					<span class=pr-4>Favorite book of the Bible?</span>
+					<Button on:click={() => alert('Saved')} danger>Save changes</Button>
+				</div>
+
+				Job
+			</Details>
+
+			<h3>Form</h3>
+			<p>Intended to remove some biolerplate and provide some consistent layout</p>
+			<Form>
+				<Input label=default />
+
+				<Button>primary</Button>
+			</Form>
+
+			<h3>Input</h3>
+			<Input label=default />
+			<Input label=password type=password />
+			<Input label=value bind:value /> <span>{value}</span>
+
+			<h3>PageHeader</h3>
+			<PageHeader>Simple</PageHeader>
+			<PageHeader>
+				<span class='flex justify-between items-center'>
+					elements spread across header
+
+					<a href='/'>link to this page</a>
+				</span>
+			</PageHeader>
+		</section>
+	</div>
+
+	<div class='drawer-side not-prose'>
+		<label for=left_drawer class=drawer-overlay />
+
+		<ul class='menu p-4 w-fit bg-base-100 text-base-content'>
+			<li class='menu-title grid grid-cols-3'>
+				<span on:click={ () => sort_items_by('name') } on:keydown={ () => sort_items_by('name') } class='cursor-pointer w-fit'>Name</span>
+				<span on:click={ () => sort_items_by('prop') } on:keydown={ () => sort_items_by('name') } class='cursor-pointer w-fit'>Prop</span>
+				<span on:click={ () => sort_items_by('number') } on:keydown={ () => sort_items_by('name') } class='cursor-pointer w-fit'>Number</span>
```

Comment:
@longrunningprocess I guess this should be `on:keydown={ () => sort_items_by('number')`. Same on line above it.

Reply:
Seems like they should always be the same. Can we create something like an angular directive to make them both the same?

Reply:
we could create what Svelte calls an action to watch both events and then do the same thing but this should be a rare case, i.e., watching a click on a non-interactive element... maybe I should just make the `a`'s.  This was for a11y's sake.

Reply:
this is probably the better solution all around:  [decided on better mark-up](https://github.com/sillsdev/web-languageforge/commit/d7f8bfc76d3f89bb825c8165f9874a27decacc40)